test/controllers/supplementary/ConfirmationPageControllerSpec.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.supplementary
+
+import base.CustomExportsBaseSpec
+import play.api.mvc.AnyContentAsEmpty
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+
+class ConfirmationPageControllerSpec extends CustomExportsBaseSpec {
+
+  private val lrn = "1234567890"
+  private val conversationId = "12QW-34ER-56TY-78UI-90OP"
+
+  override protected def getRequest(
+    uri: String,
+    headers: Map[String, String] = Map.empty
```

Comment:
You don't use `headers` here. I'm not sure that we need to override `getRequest` method. We can create the method `getRequestWithFlash` or something like this and use `getRequest` method from `CustomExportsBaseSpec`.

I'm not sure that this case is good to use override not create a new method